tests/test_correlatednoise.py

```diff
@@ -36,14 +36,6 @@
 smallim_size = 16 # size of image when we test correlated noise properties using small inputs
 smallim_size_odd = 17 # odd-sized version of the above for odd/even relevant tests (e.g. draw)
 largeim_size = 12 * smallim_size # ditto, but when we need a larger image
-xlargeim_size =long(np.ceil(1.41421356 * largeim_size)) # sometimes, for precision tests, we 
-                                                        # need a very large image that will 
-                                                        # fit a large image within it, even if 
-                                                        # rotated
-
-# Decimals for comparison (one for fine detail, another for comparing stochastic quantities)
-decimal_approx = 2
-decimal_precise = 7
```

Comment:
What was the motivation for taking these out of module scope and into the various tests as hard coded numbers?  I found it useful in the old days to be able to modify them, in one place only, to better check for breakages.  Having to search and replace for `2` or `7` seems like a bigger pain.